.travis.yml

```diff
@@ -84,10 +84,12 @@ script:
   - blt deploy:build
   # Add Drupal VM config to repo.
   - blt vm:init
+  # Run the 'blt-project' phpunit tests.
+  - phpunit vendor/acquia/blt/tests/phpunit --group blt-project
```

Comment:
You're right, they do appear to be run, I missed them when inspecting the test output earlier. I'll take that back out.